sudo/settings.py

```diff
@@ -45,3 +45,6 @@
 # The name of the session attribute used to preserve the redirect destination
 # between the original page request and successful sudo login.
 REDIRECT_TO_FIELD_NAME = getattr(settings, 'SUDO_REDIRECT_TO_FIELD_NAME', 'sudo_redirect_to')
+
+# The name of the view function for reversing
+SUDO_VIEW_NAME = getattr(settings, 'SUDO_VIEW_NAME', 'sudo.views.sudo')
```

Comment:
I think this is a better answer for the `getsentry` situation since we could use a full absolute url and avoid `reverse()` in our case, if I'm understanding the concern correctly.